config/aws_zf2.local.php.dist

```diff
@@ -8,6 +8,7 @@
 
 return [
     'aws_zf2' => [
+        'zend_framework_version' => '3.0'
```

Comment:
Instead of doing this, let's bump the entire version instead, after this PR merged in, I'll tag a new release for this :)